keymaps/templates/Apple Mail.cson

```diff
@@ -4,16 +4,39 @@
 # http://dashkards.com/img/dashkard-mail.png
 
 'body':
+
+  ### Jumping ###
+  'cmd-1' : 'navigation:go-to-inbox'
+  'cmd-2' : 'navigation:go-to-starred'
+  'cmd-3' : 'navigation:go-to-sent'
+  'cmd-4' : 'navigation:go-to-drafts'
+  'cmd-5' : 'navigation:go-to-all'
+  'cmd-6' : 'navigation:go-to-contacts'
+  'cmd-7' : 'navigation:go-to-tasks'
+  'cmd-8' : 'navigation:go-to-label'
+
+    ### Threadlist selection ###
+  'cmd-a': 'multiselect-list:select-all'
+
+
+  ### Application ###
+
+  'cmdopt-f': 'application:focus-search'
+
+
   # Triage
   'cmdctrl-r': 'application:reply'
   'cmdctrl-shift-r': 'application:reply-all'
   'cmdctrl-shift-f': 'application:forward'
   'cmdctrl-shift-j': 'application:report-as-spam'
   'cmdctrl-shift-u': 'application:mark-as-unread'
+  'cmdctrl-a': 'application:archive-item'
```

Comment:
This is a little silly, but technically `cmdctrl` maps to either "cmd" or "control" depending on the user's platform, so for the shortcut cmd+ctrl+a, it actually needs to be `cmd-ctrl-a`. (Since cmdctrl + ctrl would just be ctrl)